src/processing/script/index.js

```diff
@@ -152,9 +152,10 @@ export function processScript (src, withHeader) {
     withHeader = withHeader && !isObject && !isArrayDataScript(ast);
 
     var changes   = transform(ast);
+    var isStrict  = hasStrictDirective(ast);
```

Comment:
I still don't understand what and why we're trying to do here. Why we need to insert `use strict`? Why we need to insert it in the header?